src/integrationTest/java/uk/gov/hmcts/darts/audio/controller/AudioControllerPostAdminApproveMediaMarkedForDeletionIntTest.java

```diff
@@ -114,6 +121,13 @@ void postAdminApproveMediaMarkedForDeletionShouldReturnSuccess() throws Exceptio
         assertEquals(superAdminUser.getId(), actionResponse.getMarkedForManualDeletionById());
         assertTrue(actionResponse.getIsMarkedForManualDeletion());
 
+        List<AuditEntity> caseExpiredAuditEntries = dartsDatabase.getAuditRepository()
+            .findAll((Specification<AuditEntity>) (root, query, criteriaBuilder) -> criteriaBuilder.and(
+                criteriaBuilder.equal(root.get(AuditEntity_.additionalData), String.valueOf(adminActionEntity.getId())),
+                criteriaBuilder.equal(root.get(AuditEntity_.auditActivity).get("id"), AuditActivity.MANUAL_DELETION.getId())
+            ));
+
+        assertFalse(caseExpiredAuditEntries.isEmpty());
```

Comment:
Should validate the content of the audit entry

Reply:
Assumed this was good enough seeing as all we control was the activity and the additional data. The query looks specifically for those values so would return an empty list otherwise.

Reply:
Is this approach not consistent with the approach taken https://github.com/hmcts/darts-api/pull/2053

Reply:
Added additional assertions